mainnet/setup-erigon-rpc.md

```diff
@@ -12,7 +12,7 @@ cdk-erigon is a fork of Erigon, optimized for syncing with the XLayer network.
 ``` bash
 git clone https://github.com/0xPolygonHermez/cdk-erigon.git
 cd cdk-erigon
-git checkout v1.2.15.3 #[checkout the latest release version]
+git checkout v2.61.0 #[checkout the latest release version]
```

Comment:
For the `git clone`, should we be deploying our forked version, not the upstream?